eng/SourceBuild.props

```diff
@@ -0,0 +1,9 @@
+
+<Project>
+
+  <PropertyGroup>
+    <GitHubRepositoryName>symreader</GitHubRepositoryName>
```

Comment:
><GitHubRepositoryName>symreader</GitHubRepositoryName> [](start = 4, length = 54)

Can't this be inferred from git info?

Reply:
Potentially, but it doesn't seem like something worth implementing right now. There are risks, and it would be easy to fix incrementally later on. I've filed https://github.com/dotnet/source-build/issues/2029 with some more info and to track this.